diff -urN phpIpacStats-0.4.1.org/config.php phpIpacStats-0.4.1/config.php
--- phpIpacStats-0.4.1.org/config.php	Tue Aug 13 23:25:43 2002
+++ phpIpacStats-0.4.1/config.php	Tue Aug 13 23:26:05 2002
@@ -16,7 +16,7 @@
 // You can specify the real config-file (typically in /etc/...) or you make 
 // a copy and comment out rules which you do not want to be displayed images
 // for. 
-$ipac_conffile	    =	'ipac.conf';
+$ipac_conffile	    =	'/etc/ipac-ng/ipac.conf';
 
 // $ipacsum_executable specifies the path to your ipacsum program. you can 
 // determine this path by executing one of the following commands inside 
