diff -urN phpIpacStats-0.5.org/config.php phpIpacStats-0.5/config.php
--- phpIpacStats-0.5.org/config.php	2003-06-19 22:58:41.000000000 +0200
+++ phpIpacStats-0.5/config.php	2003-06-19 22:59:44.000000000 +0200
@@ -18,7 +18,7 @@
 // for. 
 // Usually either /etc/ipac-ng/ipac.conf or /etc/ipac-ng/rules.conf
 // For ipac-ng 1.23 and above it may be named rules.conf per default
-$ipac_conffile	    =	'/etc/ipac-ng/rules.conf';
+$ipac_conffile	    =	'/etc/ipac-ng/ipac.conf';
 
 // $ipacsum_executable specifies the path to your ipacsum program. you can 
 // determine this path by executing one of the following commands inside 
